packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -137,6 +138,7 @@ class PlotlyDivGraph extends CoreElement {
       if (_frames.containsKey(xPosition)) {
         final TimelineFrame timelineFrame = _frames[xPosition];
         framesBarChart.setSelected(timelineFrame);
+        gaSelect(gaTimeline, gaTimelineFrame);
```

Comment:
nit: maybe move this to be inside of the `setSelected` method so that if we ever select a frame bar from somewhere else, we will still capture the log

Reply:
Done and it returns two custom metrics gpu_duration and ui_duration too.